logistics/scripts/modules/settings/settings-management.psm1

```diff
@@ -322,11 +324,81 @@ function New-DevelopmentAppSettings([hashtable] $Settings = @{ }) {
         New-JsonFile $newDevelopmentSettingsPath $newDevelopmentSettings -Overwrite
         $newSettingsFiles += $newDevelopmentSettingsPath
 
-        $credentialSettings = Merge-Hashtables @{ }, (Get-CredentialSettingsByProject)[$project]
+        $credentialSettings = Merge-Hashtables @{ }, $credentialSettingsByProject[$project]
         $newUserSettingsPath = Join-Path $projectPath "appsettings.user.json"
-        New-JsonFile $newUserSettingsPath $credentialSettings
+        New-JsonFile $newUserSettingsPath $credentialSettings -Overwrite
         $newSettingsFiles += $newUserSettingsPath
     }
 
     return $newSettingsFiles
 }
+
+function Format-Json {
+    <#
```

Comment:
Found this function in stackoverflow to format JSON files.  If you have another option, please let me know.

Reply:
Very interesting